<!--yml
category: 未分类
date: 2024-05-27 14:49:49
-->

# The origins of the Guinness stout yeast | Communications Biology

> 来源：[https://www.nature.com/articles/s42003-023-05587-3](https://www.nature.com/articles/s42003-023-05587-3)

1.  Diamond, J. Evolution, consequences and future of plant and animal domestication. *Nature* **418**, 700–707 (2002).

    [Article](https://doi.org/10.1038%2Fnature01019)  [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD38XlvVyltbY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=12167878)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Evolution%2C%20consequences%20and%20future%20of%20plant%20and%20animal%20domestication&journal=Nature&doi=10.1038%2Fnature01019&volume=418&pages=700-707&publication_year=2002&author=Diamond%2CJ) 

2.  Steensels, J., Gallone, B., Voordeckers, K. & Verstrepen, K. J. Domestication of industrial microbes. *Curr. Biol.* **29**, R381–R393 (2019).

    [Article](https://doi.org/10.1016%2Fj.cub.2019.04.025)  [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtVSnu7%2FN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31112692)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Domestication%20of%20industrial%20microbes&journal=Curr.%20Biol.&doi=10.1016%2Fj.cub.2019.04.025&volume=29&pages=R381-R393&publication_year=2019&author=Steensels%2CJ&author=Gallone%2CB&author=Voordeckers%2CK&author=Verstrepen%2CKJ) 

3.  Peter, J. et al. Genome evolution across 1011 Saccharomyces cerevisiae isolates. *Nature* **556**, 339–344 (2018).

    [Article](https://doi.org/10.1038%2Fs41586-018-0030-5)  [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1cXosVCgt70%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29643504)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6784862)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome%20evolution%20across%201011%20Saccharomyces%20cerevisiae%20isolates&journal=Nature&doi=10.1038%2Fs41586-018-0030-5&volume=556&pages=339-344&publication_year=2018&author=Peter%2CJ) 

4.  Gallone, B. et al. Domestication and divergence of Saccharomyces cerevisiae beer yeasts. *Cell* **166**, 1397–1410 (2016).

    [Article](https://doi.org/10.1016%2Fj.cell.2016.08.020)  [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC28XhsFSltL%2FJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27610566)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5018251)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Domestication%20and%20divergence%20of%20Saccharomyces%20cerevisiae%20beer%20yeasts&journal=Cell&doi=10.1016%2Fj.cell.2016.08.020&volume=166&pages=1397-1410&publication_year=2016&author=Gallone%2CB) 

5.  Botstein, D. & Fink, G. R. Yeast: an experimental organism for 21st Century biology. *Genetics* **189**, 695–704 (2011).

    [Article](https://doi.org/10.1534%2Fgenetics.111.130765)  [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC38XitFKgu7c%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22084421)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3213361)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Yeast%3A%20an%20experimental%20organism%20for%2021st%20Century%20biology&journal=Genetics&doi=10.1534%2Fgenetics.111.130765&volume=189&pages=695-704&publication_year=2011&author=Botstein%2CD&author=Fink%2CGR) 

6.  Johnston, M. Genome sequencing: the complete code for a eukaryotic cell. *Curr. Biol.* **6**, 500–503 (1996).

    [Article](https://doi.org/10.1016%2FS0960-9822%2802%2900526-2)  [CAS](/articles/cas-redirect/1:CAS:528:DyaK28XjtFCrsbs%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=8805271)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genome%20sequencing%3A%20the%20complete%20code%20for%20a%20eukaryotic%20cell&journal=Curr.%20Biol.&doi=10.1016%2FS0960-9822%2802%2900526-2&volume=6&pages=500-503&publication_year=1996&author=Johnston%2CM) 

7.  Peris, D. et al. Macroevolutionary diversity of traits and genomes in the model yeast genus Saccharomyces. *Nat. Commun.* **14**, 690 (2023).

    [Article](https://doi.org/10.1038%2Fs41467-023-36139-2)  [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3sXjtVegt78%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36755033)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9908912)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Macroevolutionary%20diversity%20of%20traits%20and%20genomes%20in%20the%20model%20yeast%20genus%20Saccharomyces&journal=Nat.%20Commun.&doi=10.1038%2Fs41467-023-36139-2&volume=14&publication_year=2023&author=Peris%2CD) 

8.  Gonçalves, M. et al. Distinct domestication trajectories in top-fermenting beer yeasts and wine yeasts. *Curr. Biol.* **26**, 2750–2761 (2016).

    [Article](https://doi.org/10.1016%2Fj.cub.2016.08.040)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27720622)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Distinct%20domestication%20trajectories%20in%20top-fermenting%20beer%20yeasts%20and%20wine%20yeasts&journal=Curr.%20Biol.&doi=10.1016%2Fj.cub.2016.08.040&volume=26&pages=2750-2761&publication_year=2016&author=Gon%C3%A7alves%2CM) 

9.  Fay, J. C. et al. A polyploid admixed origin of beer yeasts derived from European and Asian wine populations. *PLoS Biol.* **17**, e3000147 (2019).

    [Article](https://doi.org/10.1371%2Fjournal.pbio.3000147)  [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtFemu7nF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30835725)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6400334)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20polyploid%20admixed%20origin%20of%20beer%20yeasts%20derived%20from%20European%20and%20Asian%20wine%20populations&journal=PLoS%20Biol.&doi=10.1371%2Fjournal.pbio.3000147&volume=17&publication_year=2019&author=Fay%2CJC) 

10.  Abou Saada, O. et al. Phased polyploid genomes provide deeper insight into the multiple origins of domesticated Saccharomyces cerevisiae beer yeasts. *Curr. Biol.* **32**, 1350–1361 (2022).

    [Article](https://doi.org/10.1016%2Fj.cub.2022.01.068)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Phased%20polyploid%20genomes%20provide%20deeper%20insight%20into%20the%20multiple%20origins%20of%20domesticated%20Saccharomyces%20cerevisiae%20beer%20yeasts&journal=Curr.%20Biol.&doi=10.1016%2Fj.cub.2022.01.068&volume=32&pages=1350-1361&publication_year=2022&author=Abou%20Saada%2CO) 

11.  Boulton, C. & Quain, D. *Brewing Yeast and Fermentation* (John Wiley & Sons, 2008).

12.  Preiss, R., Tyrawa, C., Krogerus, K., Garshol, L. M. & Van Der Merwe, G. Traditional Norwegian Kveik are a genetically distinct group of domesticated Saccharomyces cerevisiae brewing yeasts. *Front. Microbiol.* **9**, 2137 (2018).

    [Article](https://doi.org/10.3389%2Ffmicb.2018.02137)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30258422)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6145013)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Traditional%20Norwegian%20Kveik%20are%20a%20genetically%20distinct%20group%20of%20domesticated%20Saccharomyces%20cerevisiae%20brewing%20yeasts&journal=Front.%20Microbiol.&doi=10.3389%2Ffmicb.2018.02137&volume=9&publication_year=2018&author=Preiss%2CR&author=Tyrawa%2CC&author=Krogerus%2CK&author=Garshol%2CLM&author=Merwe%2CG) 

13.  GDB/FN05/0002 Articles of Association. Two copies of the Memorandum and Articles of Association of Arthur Guinness Son and Company, *Guinness Archives.* (1886).

14.  Legras, J. L. & Karst, F. Optimisation of interdelta analysis for Saccharomyces cerevisiae strain characterisation. *FEMS Microbiol. Lett.* **221**, 249–255 (2003).

    [Article](https://doi.org/10.1016%2FS0378-1097%2803%2900205-2)  [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD3sXjtlSlu7w%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=12725935)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Optimisation%20of%20interdelta%20analysis%20for%20Saccharomyces%20cerevisiae%20strain%20characterisation&journal=FEMS%20Microbiol.%20Lett.&doi=10.1016%2FS0378-1097%2803%2900205-2&volume=221&pages=249-255&publication_year=2003&author=Legras%2CJL&author=Karst%2CF) 

15.  Stamatakis, A. RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies. *Bioinformatics* **30**, 1312–1313 (2014).

    [Article](https://doi.org/10.1093%2Fbioinformatics%2Fbtu033)  [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC2cXmvFCjsbc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24451623)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3998144)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=RAxML%20version%208%3A%20a%20tool%20for%20phylogenetic%20analysis%20and%20post-analysis%20of%20large%20phylogenies&journal=Bioinformatics&doi=10.1093%2Fbioinformatics%2Fbtu033&volume=30&pages=1312-1313&publication_year=2014&author=Stamatakis%2CA) 

16.  Yu, G., Lam, T. T. Y., Zhu, H. & Guan, Y. Two methods for mapping and visualizing associated data on phylogeny using ggtree. *Mol. Biol. Evol.* **35**, 3041–3043 (2018).

    [Article](https://doi.org/10.1093%2Fmolbev%2Fmsy194)  [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtFKntrbF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30351396)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6278858)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Two%20methods%20for%20mapping%20and%20visualizing%20associated%20data%20on%20phylogeny%20using%20ggtree&journal=Mol.%20Biol.%20Evol.&doi=10.1093%2Fmolbev%2Fmsy194&volume=35&pages=3041-3043&publication_year=2018&author=Yu%2CG&author=Lam%2CTTY&author=Zhu%2CH&author=Guan%2CY) 

17.  Raj, A., Stephens, M. & Pritchard, J. K. fastSTRUCTURE: variational inference of population structure in large SNP data sets. *Genetics* **197**, 573–589 (2014).

    [Article](https://doi.org/10.1534%2Fgenetics.114.164350)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24700103)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4063916)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=fastSTRUCTURE%3A%20variational%20inference%20of%20population%20structure%20in%20large%20SNP%20data%20sets&journal=Genetics&doi=10.1534%2Fgenetics.114.164350&volume=197&pages=573-589&publication_year=2014&author=Raj%2CA&author=Stephens%2CM&author=Pritchard%2CJK) 

18.  Salazar, A. N. & Abeel, T., Alpaca: a kmer-based approach for investigating mosaic structures in microbial genomes. Preprint at *bioRxiv* [https://doi.org/10.1101/551234](https://doi.org/10.1101/551234) (2019).

19.  Salazar, A. N. et al. Nanopore sequencing enables near-complete de novo assembly of Saccharomyces cerevisiae reference strain CEN. PK113-7D. *FEMS Yeast Res.* **17**, p.fox074 (2017).

20.  Nijkamp, J. F. et al. De novo sequencing, assembly and analysis of the genome of the laboratory strain Saccharomyces cerevisiae CEN. PK113-7D, a model for modern industrial biotechnology. *Microb. Cell Factories* **11**, 1–17 (2012).

    [Article](https://link.springer.com/doi/10.1186/1475-2859-11-36)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=De%20novo%20sequencing%2C%20assembly%20and%20analysis%20of%20the%20genome%20of%20the%20laboratory%20strain%20Saccharomyces%20cerevisiae%20CEN.%20PK113-7D%2C%20a%20model%20for%20modern%20industrial%20biotechnology&journal=Microb.%20Cell%20Factories&doi=10.1186%2F1475-2859-11-36&volume=11&pages=1-17&publication_year=2012&author=Nijkamp%2CJF) 

21.  Kolmogorov, M. et al. metaFlye: scalable long-read metagenome assembly using repeat graphs. *Nat. Methods* **17**, 1103–1110 (2020).

    [Article](https://doi.org/10.1038%2Fs41592-020-00971-x)  [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3cXhvF2jtb7I)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33020656)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10699202)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=metaFlye%3A%20scalable%20long-read%20metagenome%20assembly%20using%20repeat%20graphs&journal=Nat.%20Methods&doi=10.1038%2Fs41592-020-00971-x&volume=17&pages=1103-1110&publication_year=2020&author=Kolmogorov%2CM) 

22.  Morard, M. et al. Aneuploidy and ethanol tolerance in Saccharomyces cerevisiae. *Front. Genet.* **10**, 82 (2019).

    [Article](https://doi.org/10.3389%2Ffgene.2019.00082)  [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtlWms7%2FF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30809248)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6379819)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Aneuploidy%20and%20ethanol%20tolerance%20in%20Saccharomyces%20cerevisiae&journal=Front.%20Genet.&doi=10.3389%2Ffgene.2019.00082&volume=10&publication_year=2019&author=Morard%2CM) 

23.  Bilinski, C. A. & Casey, G. P. Developments in sporulation and breeding of brewer’s yeast. *Yeast* **5**, 429–438 (1989).

    [Article](https://doi.org/10.1002%2Fyea.320050603)  [CAS](/articles/cas-redirect/1:CAS:528:DyaK3cXhvFykug%3D%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Developments%20in%20sporulation%20and%20breeding%20of%20brewer%E2%80%99s%20yeast&journal=Yeast&doi=10.1002%2Fyea.320050603&volume=5&pages=429-438&publication_year=1989&author=Bilinski%2CCA&author=Casey%2CGP) 

24.  Briggs, D. E., Brookes, P. A., Boulton, C. A. & Stevens, R. *Brewing: Science and Practice* Vol. 108 (Woodhead Publ., 2004).

25.  Charron, M. J., Read, E., Haut, S. R. & Michels, C. A. Molecular evolution of the telomere-associated MAL loci of Saccharomyces. *Genetics* **122**, 307–316 (1989).

    [Article](https://doi.org/10.1093%2Fgenetics%2F122.2.307)  [CAS](/articles/cas-redirect/1:CAS:528:DyaL1MXltFSjt78%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=2548922)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1203703)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Molecular%20evolution%20of%20the%20telomere-associated%20MAL%20loci%20of%20Saccharomyces&journal=Genetics&doi=10.1093%2Fgenetics%2F122.2.307&volume=122&pages=307-316&publication_year=1989&author=Charron%2CMJ&author=Read%2CE&author=Haut%2CSR&author=Michels%2CCA) 

26.  Naumov, G. I., Naumova, E. S. & Michels, C. A. Genetic variation of the repeated MAL loci in natural populations of Saccharomyces cerevisiae and Saccharomyces paradoxus. *Genetics* **136**, 803–812 (1994).

    [Article](https://doi.org/10.1093%2Fgenetics%2F136.3.803)  [CAS](/articles/cas-redirect/1:CAS:528:DyaK2MXpsFWq)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=8005435)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1205886)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genetic%20variation%20of%20the%20repeated%20MAL%20loci%20in%20natural%20populations%20of%20Saccharomyces%20cerevisiae%20and%20Saccharomyces%20paradoxus&journal=Genetics&doi=10.1093%2Fgenetics%2F136.3.803&volume=136&pages=803-812&publication_year=1994&author=Naumov%2CGI&author=Naumova%2CES&author=Michels%2CCA) 

27.  Meilgaard, M. C., Reid, D. S. & Wyborski, K. A. Reference standards for beer flavor terminology system. *J. Am. Soc. Brew. Chem.* **40**, 119–128 (1982).

    [CAS](/articles/cas-redirect/1:CAS:528:DyaL3sXltFyltg%3D%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Reference%20standards%20for%20beer%20flavor%20terminology%20system&journal=J.%20Am.%20Soc.%20Brew.%20Chem.&volume=40&pages=119-128&publication_year=1982&author=Meilgaard%2CMC&author=Reid%2CDS&author=Wyborski%2CKA) 

28.  Pires, E. J., Teixeira, J. A., Brányik, T. & Vicente, A. A. Yeast: the soul of beer’s aroma—a review of flavour-active esters and higher alcohols produced by the brewing yeast. *Appl. Microbiol. Biotechnol.* **98**, 1937–1949 (2014).

    [Article](https://link.springer.com/doi/10.1007/s00253-013-5470-0)  [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC2cXhvVaquw%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24384752)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Yeast%3A%20the%20soul%20of%20beer%E2%80%99s%20aroma%E2%80%94a%20review%20of%20flavour-active%20esters%20and%20higher%20alcohols%20produced%20by%20the%20brewing%20yeast&journal=Appl.%20Microbiol.%20Biotechnol.&doi=10.1007%2Fs00253-013-5470-0&volume=98&pages=1937-1949&publication_year=2014&author=Pires%2CEJ&author=Teixeira%2CJA&author=Br%C3%A1nyik%2CT&author=Vicente%2CAA) 

29.  Lilly, M., Bauer, F. F., Styger, G., Lambrechts, M. G. & Pretorius, I. S. The effect of increased branched-chain amino acid transaminase activity in yeast on the production of higher alcohols and on the flavour profiles of wine and distillates. *FEMS Yeast Res.* **6**, 726–743 (2006).

    [Article](https://doi.org/10.1111%2Fj.1567-1364.2006.00057.x)  [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD28Xotlymsbc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16879424)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20effect%20of%20increased%20branched-chain%20amino%20acid%20transaminase%20activity%20in%20yeast%20on%20the%20production%20of%20higher%20alcohols%20and%20on%20the%20flavour%20profiles%20of%20wine%20and%20distillates&journal=FEMS%20Yeast%20Res.&doi=10.1111%2Fj.1567-1364.2006.00057.x&volume=6&pages=726-743&publication_year=2006&author=Lilly%2CM&author=Bauer%2CFF&author=Styger%2CG&author=Lambrechts%2CMG&author=Pretorius%2CIS) 

30.  Styger, G., Jacobson, D., Prior, B. A. & Bauer, F. F. Genetic analysis of the metabolic pathways responsible for aroma metabolite production by Saccharomyces cerevisiae. *Appl. Microbiol. Biotechnol.* **97**, 4429–4442 (2013).

    [Article](https://link.springer.com/doi/10.1007/s00253-012-4522-1)  [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC3sXntFyrtb4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23111598)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genetic%20analysis%20of%20the%20metabolic%20pathways%20responsible%20for%20aroma%20metabolite%20production%20by%20Saccharomyces%20cerevisiae&journal=Appl.%20Microbiol.%20Biotechnol.&doi=10.1007%2Fs00253-012-4522-1&volume=97&pages=4429-4442&publication_year=2013&author=Styger%2CG&author=Jacobson%2CD&author=Prior%2CBA&author=Bauer%2CFF) 

31.  Gibson, B., Vidgren, V., Peddinti, G. & Krogerus, K. Diacetyl control during brewery fermentation via adaptive laboratory engineering of the lager yeast Saccharomyces pastorianus. *J. Ind. Microbiol. Biotechnol.* **45**, 1103–1112 (2018).

    [Article](https://link.springer.com/doi/10.1007/s10295-018-2087-4)  [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1cXhvV2ns7zI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30306366)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Diacetyl%20control%20during%20brewery%20fermentation%20via%20adaptive%20laboratory%20engineering%20of%20the%20lager%20yeast%20Saccharomyces%20pastorianus&journal=J.%20Ind.%20Microbiol.%20Biotechnol.&doi=10.1007%2Fs10295-018-2087-4&volume=45&pages=1103-1112&publication_year=2018&author=Gibson%2CB&author=Vidgren%2CV&author=Peddinti%2CG&author=Krogerus%2CK) 

32.  Krogerus, K. & Gibson, B. R. 125th anniversary review: diacetyl and its control during brewery fermentation. *J. Inst. Brew.* **119**, 86–97 (2013).

    [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC3sXhsFGgurnM)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=125th%20anniversary%20review%3A%20diacetyl%20and%20its%20control%20during%20brewery%20fermentation&journal=J.%20Inst.%20Brew.&volume=119&pages=86-97&publication_year=2013&author=Krogerus%2CK&author=Gibson%2CBR) 

33.  Cullin, C., Baudin‐Baillieu, A., Guillemet, E. & Ozier‐Kalogeropoulos, O. Functional analysis of YCL09C: evidence for a role as the regulatory subunit of acetolactate synthase. *Yeast* **12**, 1511–1518 (1996).

    [Article](https://doi.org/10.1002%2F%28SICI%291097-0061%28199612%2912%3A15%3C1511%3A%3AAID-YEA41%3E3.0.CO%3B2-B)  [CAS](/articles/cas-redirect/1:CAS:528:DyaK2sXitVCjsA%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=8972574)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Functional%20analysis%20of%20YCL09C%3A%20evidence%20for%20a%20role%20as%20the%20regulatory%20subunit%20of%20acetolactate%20synthase&journal=Yeast&doi=10.1002%2F%28SICI%291097-0061%28199612%2912%3A15%3C1511%3A%3AAID-YEA41%3E3.0.CO%3B2-B&volume=12&pages=1511-1518&publication_year=1996&author=Cullin%2CC&author=Baudin%E2%80%90Baillieu%2CA&author=Guillemet%2CE&author=Ozier%E2%80%90Kalogeropoulos%2CO) 

34.  Falco, S. C., Dumas, K. S. & Livak, K. J. Nucleotide sequence of the yeast ILV2 gene which encodes acetolactate synthase. *Nucleic acids Res.* **13**, 4011–4027 (1985).

    [Article](https://doi.org/10.1093%2Fnar%2F13.11.4011)  [CAS](/articles/cas-redirect/1:CAS:528:DyaL2MXkvValsr8%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=2989783)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC341293)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Nucleotide%20sequence%20of%20the%20yeast%20ILV2%20gene%20which%20encodes%20acetolactate%20synthase&journal=Nucleic%20acids%20Res.&doi=10.1093%2Fnar%2F13.11.4011&volume=13&pages=4011-4027&publication_year=1985&author=Falco%2CSC&author=Dumas%2CKS&author=Livak%2CKJ) 

35.  Heidlas, J. & Tressl, R. Purification and properties of two oxidoreductases catalyzing the enantioselective reduction of diacetyl and other diketones from baker’s yeast. *Eur. J. Biochem.* **188**, 165–174 (1990).

    [Article](https://doi.org/10.1111%2Fj.1432-1033.1990.tb15384.x)  [CAS](/articles/cas-redirect/1:CAS:528:DyaK3cXhsFCitLY%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=2180695)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Purification%20and%20properties%20of%20two%20oxidoreductases%20catalyzing%20the%20enantioselective%20reduction%20of%20diacetyl%20and%20other%20diketones%20from%20baker%E2%80%99s%20yeast&journal=Eur.%20J.%20Biochem.&doi=10.1111%2Fj.1432-1033.1990.tb15384.x&volume=188&pages=165-174&publication_year=1990&author=Heidlas%2CJ&author=Tressl%2CR) 

36.  Katz, M., Hahn-Hägerdal, B. & Gorwa-Grauslund, M. F. Screening of two complementary collections of Saccharomyces cerevisiae to identify enzymes involved in stereo-selective reductions of specific carbonyl compounds: an alternative to protein purification. *Enzym. Microb. Technol.* **33**, 163–172 (2003).

    [Article](https://doi.org/10.1016%2FS0141-0229%2803%2900086-3)  [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD3sXlvVWju74%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Screening%20of%20two%20complementary%20collections%20of%20Saccharomyces%20cerevisiae%20to%20identify%20enzymes%20involved%20in%20stereo-selective%20reductions%20of%20specific%20carbonyl%20compounds%3A%20an%20alternative%20to%20protein%20purification&journal=Enzym.%20Microb.%20Technol.&doi=10.1016%2FS0141-0229%2803%2900086-3&volume=33&pages=163-172&publication_year=2003&author=Katz%2CM&author=Hahn-H%C3%A4gerdal%2CB&author=Gorwa-Grauslund%2CMF) 

37.  Brumsted, D., Lauterbach, A. F. & West, D. B. Phenolic characteristics in brewing 111\. The role of yeast. In *Proc. American Society of Brewing Chemists Annual Meeting, St Louis* 142–145 (Am. Soc. Brewing Chemists, Madison, WI, 1965).

38.  Madigan, D., McMurrough, I. & Smyth, M. R. Rapid determination of 4-vinyl guaiacol and ferulic acid in beers and worts by high-performance liquid chromatography. *J. Am. Soc. Brew. Chem.* **52**, 152–155 (1994).

    [CAS](/articles/cas-redirect/1:CAS:528:DyaK2MXivVaktrs%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Rapid%20determination%20of%204-vinyl%20guaiacol%20and%20ferulic%20acid%20in%20beers%20and%20worts%20by%20high-performance%20liquid%20chromatography&journal=J.%20Am.%20Soc.%20Brew.%20Chem.&volume=52&pages=152-155&publication_year=1994&author=Madigan%2CD&author=McMurrough%2CI&author=Smyth%2CMR) 

39.  Maillard, M. N. & Berset, C. Evolution of antioxidant activity during kilning: role of insoluble bound phenolic acids of barley and malt. *J. Agric. Food Chem.* **43**, 1789–1793 (1995).

    [Article](https://doi.org/10.1021%2Fjf00055a008)  [CAS](/articles/cas-redirect/1:CAS:528:DyaK2MXmsFCrsb4%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Evolution%20of%20antioxidant%20activity%20during%20kilning%3A%20role%20of%20insoluble%20bound%20phenolic%20acids%20of%20barley%20and%20malt&journal=J.%20Agric.%20Food%20Chem.&doi=10.1021%2Fjf00055a008&volume=43&pages=1789-1793&publication_year=1995&author=Maillard%2CMN&author=Berset%2CC) 

40.  Mukai, N., Masaki, K., Fujii, T. & Iefuji, H. Single nucleotide polymorphisms of PAD1 and FDC1 show a positive relationship with ferulic acid decarboxylation ability among industrial yeasts used in alcoholic beverage production. *J. Biosci. Bioeng.* **118**, 50–55 (2014).

    [Article](https://doi.org/10.1016%2Fj.jbiosc.2013.12.017)  [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC2cXhvVahsLo%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24507903)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Single%20nucleotide%20polymorphisms%20of%20PAD1%20and%20FDC1%20show%20a%20positive%20relationship%20with%20ferulic%20acid%20decarboxylation%20ability%20among%20industrial%20yeasts%20used%20in%20alcoholic%20beverage%20production&journal=J.%20Biosci.%20Bioeng.&doi=10.1016%2Fj.jbiosc.2013.12.017&volume=118&pages=50-55&publication_year=2014&author=Mukai%2CN&author=Masaki%2CK&author=Fujii%2CT&author=Iefuji%2CH) 

41.  Vidgren, V. & Londesborough, J. 125th anniversary review: yeast flocculation and sedimentation in brewing. *J. Inst. Brew.* **117**, 475–487 (2011).

    [Article](https://doi.org/10.1002%2Fj.2050-0416.2011.tb00495.x)  [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC38XnvVOiur4%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=125th%20anniversary%20review%3A%20yeast%20flocculation%20and%20sedimentation%20in%20brewing&journal=J.%20Inst.%20Brew.&doi=10.1002%2Fj.2050-0416.2011.tb00495.x&volume=117&pages=475-487&publication_year=2011&author=Vidgren%2CV&author=Londesborough%2CJ) 

42.  Fichtner, L., Schulze, F. & Braus, G. H. Differential Flo8p‐dependent regulation of FLO1 and FLO11 for cell–cell and cell–substrate adherence of S. cerevisiae S288C. *Mol. Microbiol.* **66**, 1276–1289 (2007).

    [Article](https://doi.org/10.1111%2Fj.1365-2958.2007.06014.x)  [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD2sXhsVeku7rM)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18001350)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2780560)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Differential%20Flo8p%E2%80%90dependent%20regulation%20of%20FLO1%20and%20FLO11%20for%20cell%E2%80%93cell%20and%20cell%E2%80%93substrate%20adherence%20of%20S.%20cerevisiae%20S288C&journal=Mol.%20Microbiol.&doi=10.1111%2Fj.1365-2958.2007.06014.x&volume=66&pages=1276-1289&publication_year=2007&author=Fichtner%2CL&author=Schulze%2CF&author=Braus%2CGH) 

43.  Stratford, M. Yeast flocculation: calcium specificity. *Yeast* **5**, 487–496 (1989).

    [Article](https://doi.org/10.1002%2Fyea.320050608)  [CAS](/articles/cas-redirect/1:CAS:528:DyaK3cXhvVyiug%3D%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Yeast%20flocculation%3A%20calcium%20specificity&journal=Yeast&doi=10.1002%2Fyea.320050608&volume=5&pages=487-496&publication_year=1989&author=Stratford%2CM) 

44.  Stratford, M. Yeast flocculation: a new perspective. *Adv. Microb. Physiol.* **33**, 1–71 (1992).

    [Article](https://doi.org/10.1016%2FS0065-2911%2808%2960215-5)  [CAS](/articles/cas-redirect/1:CAS:528:DyaK3sXhtVSgu7k%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Yeast%20flocculation%3A%20a%20new%20perspective&journal=Adv.%20Microb.%20Physiol.&doi=10.1016%2FS0065-2911%2808%2960215-5&volume=33&pages=1-71&publication_year=1992&author=Stratford%2CM) 

45.  Straver, M. H., Aar, P. C., Smit, G. & Kijne, J. W. Determinants of flocculence of brewer’s yeast during fermentation in wort. *Yeast* **9**, 527–532 (1993).

    [Article](https://doi.org/10.1002%2Fyea.320090509)  [CAS](/articles/cas-redirect/1:CAS:528:DyaK3sXkslWhsLc%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=8322515)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Determinants%20of%20flocculence%20of%20brewer%E2%80%99s%20yeast%20during%20fermentation%20in%20wort&journal=Yeast&doi=10.1002%2Fyea.320090509&volume=9&pages=527-532&publication_year=1993&author=Straver%2CMH&author=Aar%2CPC&author=Smit%2CG&author=Kijne%2CJW) 

46.  Stratford, M. Induction of flocculation in brewing yeasts by change in pH value. *FEMS Microbiol. Lett.* **136**, 13–18 (1996).

    [Article](https://doi.org/10.1111%2Fj.1574-6968.1996.tb08018.x)  [CAS](/articles/cas-redirect/1:CAS:528:DyaK28XhsFOhur4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=8919449)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Induction%20of%20flocculation%20in%20brewing%20yeasts%20by%20change%20in%20pH%20value&journal=FEMS%20Microbiol.%20Lett.&doi=10.1111%2Fj.1574-6968.1996.tb08018.x&volume=136&pages=13-18&publication_year=1996&author=Stratford%2CM) 

47.  Goossens, K. & Willaert, R. Flocculation protein structure and cell–cell adhesion mechanism in Saccharomyces cerevisiae. *Biotechnol. Lett.* **32**, 1571–1585 (2010).

    [Article](https://link.springer.com/doi/10.1007/s10529-010-0352-3)  [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC3cXht1KgsrzE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20640875)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Flocculation%20protein%20structure%20and%20cell%E2%80%93cell%20adhesion%20mechanism%20in%20Saccharomyces%20cerevisiae&journal=Biotechnol.%20Lett.&doi=10.1007%2Fs10529-010-0352-3&volume=32&pages=1571-1585&publication_year=2010&author=Goossens%2CK&author=Willaert%2CR) 

48.  Gilliland, R. B. *Proc. European Brewery Convention, Brighton* (1951).

49.  Verstrepen, K. J., Jansen, A., Lewitter, F. & Fink, G. R. Intragenic tandem repeats generate functional variability. *Nat. Genet.* **37**, 986–990 (2005).

    [Article](https://doi.org/10.1038%2Fng1618)  [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD2MXpsFWisLk%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16086015)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1462868)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Intragenic%20tandem%20repeats%20generate%20functional%20variability&journal=Nat.%20Genet.&doi=10.1038%2Fng1618&volume=37&pages=986-990&publication_year=2005&author=Verstrepen%2CKJ&author=Jansen%2CA&author=Lewitter%2CF&author=Fink%2CGR) 

50.  Watari, J., Nomura, M., Sahara, H., Koshino, S. & Keränen, S. Construction of flocculent brewer’s yeast by chromosomal integration of the yeast flocculation gene FLO1\. *J. Inst. Brew.* **100**, 73–77 (1994).

    [Article](https://doi.org/10.1002%2Fj.2050-0416.1994.tb00809.x)  [CAS](/articles/cas-redirect/1:CAS:528:DyaK2MXmtFartA%3D%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Construction%20of%20flocculent%20brewer%E2%80%99s%20yeast%20by%20chromosomal%20integration%20of%20the%20yeast%20flocculation%20gene%20FLO1&journal=J.%20Inst.%20Brew.&doi=10.1002%2Fj.2050-0416.1994.tb00809.x&volume=100&pages=73-77&publication_year=1994&author=Watari%2CJ&author=Nomura%2CM&author=Sahara%2CH&author=Koshino%2CS&author=Ker%C3%A4nen%2CS) 

51.  Lo, W. S. & Dranginis, A. FLO11, a yeast gene related to the STA genes, encodes a novel cell surface flocculin. *J. Bacteriol.* **178**, 7144–7151 (1996).

    [Article](https://doi.org/10.1128%2Fjb.178.24.7144-7151.1996)  [CAS](/articles/cas-redirect/1:CAS:528:DyaK2sXhtlWhtg%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=8955395)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC178626)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=FLO11%2C%20a%20yeast%20gene%20related%20to%20the%20STA%20genes%2C%20encodes%20a%20novel%20cell%20surface%20flocculin&journal=J.%20Bacteriol.&doi=10.1128%2Fjb.178.24.7144-7151.1996&volume=178&pages=7144-7151&publication_year=1996&author=Lo%2CWS&author=Dranginis%2CA) 

52.  Anderson, R. One yeast or two? Pure yeast and top fermentation. *Brew. Hist.* **149**, 30–38 (2012).

    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=One%20yeast%20or%20two%3F%20Pure%20yeast%20and%20top%20fermentation&journal=Brew.%20Hist.&volume=149&pages=30-38&publication_year=2012&author=Anderson%2CR) 

53.  GDB/RD02/0365\. File No 521A - Visits in Great Britain and Ireland 1902-11-21-1978-12-07\. In *Guinness Archives.*

54.  Russell, I. & Stewart, G. G. Liquid nitrogen storage of yeast cultures compared to more traditional storage methods. *J. Am. Soc. Brew. Chem.* **39**, 19–24 (1981).

    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Liquid%20nitrogen%20storage%20of%20yeast%20cultures%20compared%20to%20more%20traditional%20storage%20methods&journal=J.%20Am.%20Soc.%20Brew.%20Chem.&volume=39&pages=19-24&publication_year=1981&author=Russell%2CI&author=Stewart%2CGG) 

55.  Donnelly, D. & Hurley, J. Yeast monitoring: the Guinness experience. *Ferment* **9**, 283–286 (1996).

    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Yeast%20monitoring%3A%20the%20Guinness%20experience&journal=Ferment&volume=9&pages=283-286&publication_year=1996&author=Donnelly%2CD&author=Hurley%2CJ) 

56.  Berry, D. R. & Slaughter, J. C., Alcoholic beverage fermentations. In *Fermented beverage production*. 25-39 (Springer, Boston, MA, US, 2003).

57.  Ekberg, J. et al. Adaptive evolution of the lager brewing yeast Saccharomyces pastorianus for improved growth under hyperosmotic conditions and its influence on fermentation performance. *FEMS Yeast Res.* **13**, 335–349 (2013).

    [Article](https://doi.org/10.1111%2F1567-1364.12038)  [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC3sXntlers7g%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23414064)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Adaptive%20evolution%20of%20the%20lager%20brewing%20yeast%20Saccharomyces%20pastorianus%20for%20improved%20growth%20under%20hyperosmotic%20conditions%20and%20its%20influence%20on%20fermentation%20performance&journal=FEMS%20Yeast%20Res.&doi=10.1111%2F1567-1364.12038&volume=13&pages=335-349&publication_year=2013&author=Ekberg%2CJ) 

58.  Gibson, B. et al. Adaptive laboratory evolution of ale and lager yeasts for improved brewing efficiency and beer quality. *Annu. Rev. Food Sci. Technol.* **11**, 23–44 (2020).

    [Article](https://doi.org/10.1146%2Fannurev-food-032519-051715)  [CAS](/articles/cas-redirect/1:STN:280:DC%2BB38%2FhsV2nsw%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31951488)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Adaptive%20laboratory%20evolution%20of%20ale%20and%20lager%20yeasts%20for%20improved%20brewing%20efficiency%20and%20beer%20quality&journal=Annu.%20Rev.%20Food%20Sci.%20Technol.&doi=10.1146%2Fannurev-food-032519-051715&volume=11&pages=23-44&publication_year=2020&author=Gibson%2CB) 

59.  Albertin, W. et al. Evidence for autotetraploidy associated with reproductive isolation in Saccharomyces cerevisiae: towards a new domesticated species. *J. Evolut. Biol.* **22**, 2157–2170 (2009).

    [Article](https://doi.org/10.1111%2Fj.1420-9101.2009.01828.x)  [CAS](/articles/cas-redirect/1:STN:280:DC%2BC3c%2FhtlymsA%3D%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Evidence%20for%20autotetraploidy%20associated%20with%20reproductive%20isolation%20in%20Saccharomyces%20cerevisiae%3A%20towards%20a%20new%20domesticated%20species&journal=J.%20Evolut.%20Biol.&doi=10.1111%2Fj.1420-9101.2009.01828.x&volume=22&pages=2157-2170&publication_year=2009&author=Albertin%2CW) 

60.  Voordeckers, K. & Verstrepen, K. J. Experimental evolution of the model eukaryote Saccharomyces cerevisiae yields insight into the molecular mechanisms underlying adaptation. *Curr. Opin. Microbiol.* **28**, 1–9 (2015).

    [Article](https://doi.org/10.1016%2Fj.mib.2015.06.018)  [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC2MXhtF2mt7%2FP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26202939)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Experimental%20evolution%20of%20the%20model%20eukaryote%20Saccharomyces%20cerevisiae%20yields%20insight%20into%20the%20molecular%20mechanisms%20underlying%20adaptation&journal=Curr.%20Opin.%20Microbiol.&doi=10.1016%2Fj.mib.2015.06.018&volume=28&pages=1-9&publication_year=2015&author=Voordeckers%2CK&author=Verstrepen%2CKJ) 

61.  Yona, A. H. et al. Chromosomal duplication is a transient evolutionary solution to stress. *Proc. Natl Acad. Sci.* **109**, 21010–21015 (2012).

    [Article](https://doi.org/10.1073%2Fpnas.1211150109)  [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC3sXnt1emsg%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23197825)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3529009)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Chromosomal%20duplication%20is%20a%20transient%20evolutionary%20solution%20to%20stress&journal=Proc.%20Natl%20Acad.%20Sci.&doi=10.1073%2Fpnas.1211150109&volume=109&pages=21010-21015&publication_year=2012&author=Yona%2CAH) 

62.  Linder, R. A., Greco, J. P., Seidl, F., Matsui, T. & Ehrenreich, I. M. The stress-inducible peroxidase TSA2 underlies a conditionally beneficial chromosomal duplication in Saccharomyces cerevisiae. *G3 Genes Genomes Genet.* **7**, 3177–3184 (2017).

    [Article](https://doi.org/10.1534%2Fg3.117.300069)  [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1cXisVOrtrzO)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20stress-inducible%20peroxidase%20TSA2%20underlies%20a%20conditionally%20beneficial%20chromosomal%20duplication%20in%20Saccharomyces%20cerevisiae&journal=G3%20Genes%20Genomes%20Genet.&doi=10.1534%2Fg3.117.300069&volume=7&pages=3177-3184&publication_year=2017&author=Linder%2CRA&author=Greco%2CJP&author=Seidl%2CF&author=Matsui%2CT&author=Ehrenreich%2CIM) 

63.  Scopel, E. F., Hose, J., Bensasson, D. & Gasch, A. P. Genetic variation in aneuploidy prevalence and tolerance across Saccharomyces cerevisiae lineages. *Genetics* **217**, iyab015 (2021).

    [Article](https://doi.org/10.1093%2Fgenetics%2Fiyab015)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33734361)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8049548)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Genetic%20variation%20in%20aneuploidy%20prevalence%20and%20tolerance%20across%20Saccharomyces%20cerevisiae%20lineages&journal=Genetics&doi=10.1093%2Fgenetics%2Fiyab015&volume=217&publication_year=2021&author=Scopel%2CEF&author=Hose%2CJ&author=Bensasson%2CD&author=Gasch%2CAP) 

64.  Torres, E. M. et al. Effects of aneuploidy on cellular physiology and cell division in haploid yeast. *Science* **317**, 916–924 (2007).

    [Article](https://doi.org/10.1126%2Fscience.1142210)  [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD2sXptVaisro%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17702937)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Effects%20of%20aneuploidy%20on%20cellular%20physiology%20and%20cell%20division%20in%20haploid%20yeast&journal=Science&doi=10.1126%2Fscience.1142210&volume=317&pages=916-924&publication_year=2007&author=Torres%2CEM) 

65.  Heavner, B. D. & Price, N. D. Comparative analysis of yeast metabolic network models highlights progress, opportunities for metabolic reconstruction. *PLoS Comput. Biol.* **11**, e1004530 (2015).

    [Article](https://doi.org/10.1371%2Fjournal.pcbi.1004530)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26566239)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4643975)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Comparative%20analysis%20of%20yeast%20metabolic%20network%20models%20highlights%20progress%2C%20opportunities%20for%20metabolic%20reconstruction&journal=PLoS%20Comput.%20Biol.&doi=10.1371%2Fjournal.pcbi.1004530&volume=11&publication_year=2015&author=Heavner%2CBD&author=Price%2CND) 

66.  Pereira, R., Nielsen, J. & Rocha, I. Improving the flux distributions simulated with genome-scale metabolic models of Saccharomyces cerevisiae. *Metab. Eng. Commun.* **3**, 153–163 (2016).

    [Article](https://doi.org/10.1016%2Fj.meteno.2016.05.002)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29468121)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5779720)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Improving%20the%20flux%20distributions%20simulated%20with%20genome-scale%20metabolic%20models%20of%20Saccharomyces%20cerevisiae&journal=Metab.%20Eng.%20Commun.&doi=10.1016%2Fj.meteno.2016.05.002&volume=3&pages=153-163&publication_year=2016&author=Pereira%2CR&author=Nielsen%2CJ&author=Rocha%2CI) 

67.  Szappanos, B. et al. An integrated approach to characterize genetic interaction networks in yeast metabolism. *Nat. Genet.* **43**, 656–662 (2011).

    [Article](https://doi.org/10.1038%2Fng.846)  [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC3MXmslSrtb8%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21623372)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3125439)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=An%20integrated%20approach%20to%20characterize%20genetic%20interaction%20networks%20in%20yeast%20metabolism&journal=Nat.%20Genet.&doi=10.1038%2Fng.846&volume=43&pages=656-662&publication_year=2011&author=Szappanos%2CB) 

68.  Brochado, A. R., Andrejev, S., Maranas, C. D. & Patil, K. R. Impact of stoichiometry representation on simulation of genotype-phenotype relationships in metabolic networks. *PLoS Comput. Biol.* **8**, e1002758 (2012).

    [Article](https://doi.org/10.1371%2Fjournal.pcbi.1002758)  [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC38XhslWjsr%2FN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23133362)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3486866)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Impact%20of%20stoichiometry%20representation%20on%20simulation%20of%20genotype-phenotype%20relationships%20in%20metabolic%20networks&journal=PLoS%20Comput.%20Biol.&doi=10.1371%2Fjournal.pcbi.1002758&volume=8&publication_year=2012&author=Brochado%2CAR&author=Andrejev%2CS&author=Maranas%2CCD&author=Patil%2CKR) 

69.  Grinberg, N. F., Orhobor, O. I. & King, R. D. An evaluation of machine-learning for predicting phenotype: studies in yeast, rice, and wheat. *Mach. Learn.* **109**, 251–277 (2020).

    [Article](https://link.springer.com/doi/10.1007/s10994-019-05848-5)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32174648)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=An%20evaluation%20of%20machine-learning%20for%20predicting%20phenotype%3A%20studies%20in%20yeast%2C%20rice%2C%20and%20wheat&journal=Mach.%20Learn.&doi=10.1007%2Fs10994-019-05848-5&volume=109&pages=251-277&publication_year=2020&author=Grinberg%2CNF&author=Orhobor%2COI&author=King%2CRD) 

70.  Boulton, C. A. 4 Developments in Brewery Fermentation. *Biotechnol. Genet. Eng. Rev.* **9**, 127–181 (1991).

    [CAS](/articles/cas-redirect/1:CAS:528:DyaK3sXhtVSmtLs%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=1797013)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=4%20Developments%20in%20Brewery%20Fermentation&journal=Biotechnol.%20Genet.%20Eng.%20Rev.&volume=9&pages=127-181&publication_year=1991&author=Boulton%2CCA) 

71.  Smart, K. A. & Whisker, S. Effect of serial repitching on the fermentation properties and condition of brewing yeast. *J. Am. Soc. Brew. Chem.* **54**, 41–44 (1996).

    [CAS](/articles/cas-redirect/1:CAS:528:DyaK28XovFWjtQ%3D%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Effect%20of%20serial%20repitching%20on%20the%20fermentation%20properties%20and%20condition%20of%20brewing%20yeast&journal=J.%20Am.%20Soc.%20Brew.%20Chem.&volume=54&pages=41-44&publication_year=1996&author=Smart%2CKA&author=Whisker%2CS) 

72.  Powell, C. D. & Diacetis, A. N. Long term serial repitching and the genetic and phenotypic stability of brewer’s yeast. *J. Inst. Brew.* **113**, 67–74 (2007).

    [Article](https://doi.org/10.1002%2Fj.2050-0416.2007.tb00258.x)  [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD2sXntlOntLY%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Long%20term%20serial%20repitching%20and%20the%20genetic%20and%20phenotypic%20stability%20of%20brewer%E2%80%99s%20yeast&journal=J.%20Inst.%20Brew.&doi=10.1002%2Fj.2050-0416.2007.tb00258.x&volume=113&pages=67-74&publication_year=2007&author=Powell%2CCD&author=Diacetis%2CAN) 

73.  McMurrough, I. et al. Control of ferulic acid and 4‐vinyl guaiacol in brewing. *J. Inst. Brew.* **102**, 327–332 (1996).

    [Article](https://doi.org/10.1002%2Fj.2050-0416.1996.tb00918.x)  [CAS](/articles/cas-redirect/1:CAS:528:DyaK28XmsFCjtLk%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Control%20of%20ferulic%20acid%20and%204%E2%80%90vinyl%20guaiacol%20in%20brewing&journal=J.%20Inst.%20Brew.&doi=10.1002%2Fj.2050-0416.1996.tb00918.x&volume=102&pages=327-332&publication_year=1996&author=McMurrough%2CI) 

74.  Stewart, G. G., Russell, I. & Anstruther, A. eds. *Handbook of Brewing* (CRC Press, 2017).

75.  Knight, S., Klaere, S., Fedrizzi, B. & Goddard, M. R. Regional microbial signatures positively correlate with differential wine phenotypes: evidence for a microbial aspect to terroir. *Sci. Rep.* **5**, 1–10 (2015).

    [Article](https://doi.org/10.1038%2Fsrep14233)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Regional%20microbial%20signatures%20positively%20correlate%20with%20differential%20wine%20phenotypes%3A%20evidence%20for%20a%20microbial%20aspect%20to%20terroir&journal=Sci.%20Rep.&doi=10.1038%2Fsrep14233&volume=5&pages=1-10&publication_year=2015&author=Knight%2CS&author=Klaere%2CS&author=Fedrizzi%2CB&author=Goddard%2CMR) 

76.  Morcol, T. B., Negrin, A., Matthews, P. D. & Kennelly, E. J. Hop (Humulus lupulus L.) terroir has large effect on a glycosylated green leaf volatile but not on other aroma glycosides. *Food Chem.* **321**, 126644 (2020).

    [Article](https://doi.org/10.1016%2Fj.foodchem.2020.126644)  [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3cXmt1Kgs78%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32247886)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Hop%20%28Humulus%20lupulus%20L.%29%20terroir%20has%20large%20effect%20on%20a%20glycosylated%20green%20leaf%20volatile%20but%20not%20on%20other%20aroma%20glycosides&journal=Food%20Chem.&doi=10.1016%2Fj.foodchem.2020.126644&volume=321&publication_year=2020&author=Morcol%2CTB&author=Negrin%2CA&author=Matthews%2CPD&author=Kennelly%2CEJ) 

77.  Van Holle, A. et al. Relevance of hop terroir for beer flavour. *J. Inst. Brew.* **127**, 238–247 (2021).

    [Article](https://doi.org/10.1002%2Fjib.648)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Relevance%20of%20hop%20terroir%20for%20beer%20flavour&journal=J.%20Inst.%20Brew.&doi=10.1002%2Fjib.648&volume=127&pages=238-247&publication_year=2021&author=Holle%2CA) 

78.  Richards, M. The use of giant‐colony morphology for the differentiation of brewing yeasts. *J. Inst. Brew.* **73**, 162–166 (1967).

    [Article](https://doi.org/10.1002%2Fj.2050-0416.1967.tb03028.x)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20use%20of%20giant%E2%80%90colony%20morphology%20for%20the%20differentiation%20of%20brewing%20yeasts&journal=J.%20Inst.%20Brew.&doi=10.1002%2Fj.2050-0416.1967.tb03028.x&volume=73&pages=162-166&publication_year=1967&author=Richards%2CM) 

79.  ASBC method yeast 13\. American Society of Brewing Chemists. Report of subcommittee on differentiation of brewing yeast strains by PCR fingerprinting. *J. Am. Soc. Brew. Chem.* **66**, 256–260 (2008).

    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Report%20of%20subcommittee%20on%20differentiation%20of%20brewing%20yeast%20strains%20by%20PCR%20fingerprinting&journal=J.%20Am.%20Soc.%20Brew.%20Chem.&volume=66&pages=256-260&publication_year=2008) 

80.  Li, H. & Durbin, R. Fast and accurate short read alignment with Burrows–Wheeler transform. *Bioinformatics* **25**, 1754–1760 (2009).

    [Article](https://doi.org/10.1093%2Fbioinformatics%2Fbtp324)  [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD1MXot1Cjtbo%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19451168)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2705234)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows%E2%80%93Wheeler%20transform&journal=Bioinformatics&doi=10.1093%2Fbioinformatics%2Fbtp324&volume=25&pages=1754-1760&publication_year=2009&author=Li%2CH&author=Durbin%2CR) 

81.  Danecek, P. et al. Twelve years of SAMtools and BCFtools. *Gigascience* **10**, giab008 (2021).

    [Article](https://doi.org/10.1093%2Fgigascience%2Fgiab008)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33590861)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7931819)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Twelve%20years%20of%20SAMtools%20and%20BCFtools&journal=Gigascience&doi=10.1093%2Fgigascience%2Fgiab008&volume=10&publication_year=2021&author=Danecek%2CP) 

82.  Okonechnikov, K., Conesa, A. & García-Alcalde, F. Qualimap 2: advanced multi-sample quality control for high-throughput sequencing data. *Bioinformatics* **32**, 292–294 (2016).

    [Article](https://doi.org/10.1093%2Fbioinformatics%2Fbtv566)  [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC28XhtlaqsLzL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26428292)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Qualimap%202%3A%20advanced%20multi-sample%20quality%20control%20for%20high-throughput%20sequencing%20data&journal=Bioinformatics&doi=10.1093%2Fbioinformatics%2Fbtv566&volume=32&pages=292-294&publication_year=2016&author=Okonechnikov%2CK&author=Conesa%2CA&author=Garc%C3%ADa-Alcalde%2CF) 

83.  Alonge, M. et al. RaGOO: fast and accurate reference-guided scaffolding of draft genomes. *Genome Biol.* **20**, 1–17 (2019).

    [Article](https://link.springer.com/doi/10.1186/s13059-019-1829-6)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=RaGOO%3A%20fast%20and%20accurate%20reference-guided%20scaffolding%20of%20draft%20genomes&journal=Genome%20Biol.&doi=10.1186%2Fs13059-019-1829-6&volume=20&pages=1-17&publication_year=2019&author=Alonge%2CM) 

84.  Murigneux, V. et al. Comparison of long-read methods for sequencing and assembly of a plant genome. *GigaScience* **9**, giaa146 (2020).

    [Article](https://doi.org/10.1093%2Fgigascience%2Fgiaa146)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33347571)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7751402)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Comparison%20of%20long-read%20methods%20for%20sequencing%20and%20assembly%20of%20a%20plant%20genome&journal=GigaScience&doi=10.1093%2Fgigascience%2Fgiaa146&volume=9&publication_year=2020&author=Murigneux%2CV) 

85.  Waterhouse, R. M. et al. BUSCO applications from quality assessments to gene prediction and phylogenomics. *Mol. Biol. Evol.* **35**, 543–548 (2018).

    [Article](https://doi.org/10.1093%2Fmolbev%2Fmsx319)  [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1cXitlyjt7jP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29220515)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=BUSCO%20applications%20from%20quality%20assessments%20to%20gene%20prediction%20and%20phylogenomics&journal=Mol.%20Biol.%20Evol.&doi=10.1093%2Fmolbev%2Fmsx319&volume=35&pages=543-548&publication_year=2018&author=Waterhouse%2CRM) 

86.  Palmer, J. & Stajich, J. nextgenusfs/funannotate: funannotate v1.5.3 (Version 1.5.3). *Zenodo* [https://doi.org/10.5281/zenodo.2604804](https://doi.org/10.5281/zenodo.2604804) (2019).

    [Article](https://doi.org/10.5281%2Fzenodo.2604804)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=nextgenusfs%2Ffunannotate%3A%20funannotate%20v1.5.3%20%28Version%201.5.3%29&journal=Zenodo&doi=10.5281%2Fzenodo.2604804&publication_year=2019&author=Palmer%2CJ&author=Stajich%2CJ) 

87.  Van der Auwera, G. A. & O’Connor, B. D. *Genomics in the Cloud: Using Docker, GATK, and WDL in Terra* (O’Reilly Media, 2020).

88.  Cingolani, P. et al. A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3\. *Fly* **6**, 80–92 (2012).

    [Article](https://doi.org/10.4161%2Ffly.19695)  [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC38Xht1GmtL3E)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22728672)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3679285)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20program%20for%20annotating%20and%20predicting%20the%20effects%20of%20single%20nucleotide%20polymorphisms%2C%20SnpEff%3A%20SNPs%20in%20the%20genome%20of%20Drosophila%20melanogaster%20strain%20w1118%3B%20iso-2%3B%20iso-3&journal=Fly&doi=10.4161%2Ffly.19695&volume=6&pages=80-92&publication_year=2012&author=Cingolani%2CP) 

89.  Emms, D. M. & Kelly, S. OrthoFinder: phylogenetic orthology inference for comparative genomics. *Genome Biol.* **20**, 1–14 (2019).

    [Article](https://link.springer.com/doi/10.1186/s13059-019-1832-y)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=OrthoFinder%3A%20phylogenetic%20orthology%20inference%20for%20comparative%20genomics&journal=Genome%20Biol.&doi=10.1186%2Fs13059-019-1832-y&volume=20&pages=1-14&publication_year=2019&author=Emms%2CDM&author=Kelly%2CS) 

90.  Purcell, S. et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. *Am. J. Hum. Genet.* **81**, 559–575 (2007).

    [Article](https://doi.org/10.1086%2F519795)  [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD2sXhtVSqurrL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17701901)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1950838)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=PLINK%3A%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&journal=Am.%20J.%20Hum.%20Genet.&doi=10.1086%2F519795&volume=81&pages=559-575&publication_year=2007&author=Purcell%2CS) 

91.  Boeva, V. et al. Control-FREEC: a tool for assessing copy number and allelic content using next-generation sequencing data. *Bioinformatics* **28**, 423–425 (2012).

    [Article](https://doi.org/10.1093%2Fbioinformatics%2Fbtr670)  [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC38XhvVertbo%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22155870)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Control-FREEC%3A%20a%20tool%20for%20assessing%20copy%20number%20and%20allelic%20content%20using%20next-generation%20sequencing%20data&journal=Bioinformatics&doi=10.1093%2Fbioinformatics%2Fbtr670&volume=28&pages=423-425&publication_year=2012&author=Boeva%2CV) 

92.  Fijarczyk, A. et al. The genome sequence of the Jean-Talon strain, an archeological beer yeast from Québec, reveals traces of adaptation to specific brewing conditions. *G3 Genes Genomes Genet.* **10**, 3087–3097 (2020).

    [Article](https://doi.org/10.1534%2Fg3.120.401149)  [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXmvVWmur4%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20genome%20sequence%20of%20the%20Jean-Talon%20strain%2C%20an%20archeological%20beer%20yeast%20from%20Qu%C3%A9bec%2C%20reveals%20traces%20of%20adaptation%20to%20specific%20brewing%20conditions&journal=G3%20Genes%20Genomes%20Genet.&doi=10.1534%2Fg3.120.401149&volume=10&pages=3087-3097&publication_year=2020&author=Fijarczyk%2CA) 

93.  ASBC method yeast 7\. American Society of Brewing Chemists. Report of Subcommittee on Microbiology. *J. Am. Soc. Brew. Chem.* **43**, 16 (1985).

    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Report%20of%20Subcommittee%20on%20Microbiology&journal=J.%20Am.%20Soc.%20Brew.%20Chem.&volume=43&publication_year=1985) 

94.  Jorgensen A. & Hansen A. *Mikroorganismen der Gärungsindustrie* 7 Aufl (Verlag Hans Carl, Nürnberg, 1956).

95.  Smart, K. A., Chambers, K. M., Lambert, I. & Jenkins, C. *J. Am. Soc. Brew. Chem.* **57**, 18 (1999).

    [CAS](/articles/cas-redirect/1:CAS:528:DyaK1MXhsVyms7c%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=&journal=J.%20Am.%20Soc.%20Brew.%20Chem.&volume=57&publication_year=1999&author=Smart%2CKA&author=Chambers%2CKM&author=Lambert%2CI&author=Jenkins%2CC) 

96.  Buckee, G. K. & Mundy, A. P. Determination of vicinal diketones in beer by gas chromatography (headspace technique) – collaborative trial. *J. Inst. Brew.* **100**, 247–253 (1994).

    [Article](https://doi.org/10.1002%2Fj.2050-0416.1994.tb00820.x)  [CAS](/articles/cas-redirect/1:CAS:528:DyaK2MXhtVCisrc%3D)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Determination%20of%20vicinal%20diketones%20in%20beer%20by%20gas%20chromatography%20%28headspace%20technique%29%20%E2%80%93%20collaborative%20trial&journal=J.%20Inst.%20Brew.&doi=10.1002%2Fj.2050-0416.1994.tb00820.x&volume=100&pages=247-253&publication_year=1994&author=Buckee%2CGK&author=Mundy%2CAP) 

97.  Brewers of Europe. Phenolic off Flavour Method. *Analytica-EBC* 2.3.9.5\. (2018).

98.  dos Santos Navarro, R. D. C. et al. Optimized Descriptive Profile: a rapid methodology for sensory description. *Food Qual. Preference* **24**, 190–200 (2012).

    [Article](https://doi.org/10.1016%2Fj.foodqual.2011.10.014)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Optimized%20Descriptive%20Profile%3A%20a%20rapid%20methodology%20for%20sensory%20description&journal=Food%20Qual.%20Preference&doi=10.1016%2Fj.foodqual.2011.10.014&volume=24&pages=190-200&publication_year=2012&author=Santos%20Navarro%2CRDC) 

99.  Lawless, H. T. & Heymann, H. *Sensory Evaluation of Food: Principles and Practices* Vol. 2 (Springer, New York, 2010).